packages/block-editor/src/store/actions.js

```diff
@@ -1106,3 +1106,17 @@ export function setHasControlledInnerBlocks(
 		clientId,
 	};
 }
+
+export function addHoveredEntity( clientId ) {
```

Comment:
I feel like we want to avoid mingling the "entity" concept with the block editor. I think the idea of an "inner block controller" is similar (the "inner block controller" is more of an "entity block controller", but the entity stuff is outside of the scope of the block-editor package.)